doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -363,10 +363,31 @@ Swift is the object storage management service, Swift depends on Keystone and po
 _The Swift backends include iSCSI drives, Ceph RBD and NFS._ 
 
 #### 4.3.1.5 Neutron
-Neutron is the networking service, Neutron depends on Keystone and has services running on the control nodes and the compute nodes:
--	neutron-api
--	neutron-rpc
--	neutron-*-agent agents which runs on Compute and Network nodes
+Neutron is the networking service, Neutron depends on Keystone and has services running on the control nodes and the compute nodes. Depending upon the workloads to be hosted by the Infrastructure and the expected load on the controller node, some of the Neutron services can run on separate network node(s). Factors affecting controller node load include number of compute nodes and the concurrent number of nova API calls as well as NAT. In this case, network nodes are widely added to manage L3 traffic for overlay tenant networks and interconnection with external networks. Table 4-2 below lists the networking service components and their placement. Please note that while network nodes are listed in the table below, network nodes only deal with tenant networks and not provider networks. Also, network nodes are not required when SDN is utilized for networking.
```

Comment:
```In this case, network nodes are widely added to manage L3 traffic for overlay tenant networks and interconnection with external networks``` -> "In this case... " seems like a disconnect from the proceeding lines. Can you re-phrase it as ```In most production use-cases, separate neutron nodes are dedicated to  manage L3 traffic for overlay tenant networks and interconnection with external networks```

Reply:
@msidana The reason for not using the term "in most production use-cases" is that when SDN controllers are deployed (predominantly in production), network nodes are not needed.

Changed to, "To reduce controller node load, network nodes are widely added to manage L3 traffic for overlay tenant networks and interconnection with external networks."